### PR TITLE
Replace lodash-es with lodash

### PR DESCRIPTION
**Pull request by axelboc**
Applying some of the things I learned while working on Braggy.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -10,7 +10,8 @@ node_modules/
 /storybook-static/
 /packages/**/dist/
 
-/.env.*
+/.env.local
+/.env.*.local
```
Comment (axelboc) [inline .gitignore:14]: Not technically needed, but more accurate: ignores `.env.test.local` but not `.env.test`, for instance.

File: .prettierignore [hunk 2]
```diff
@@ -1,4 +1,3 @@
 /build/
 /storybook-static/
 /packages/**/dist/
-/package-lock.json
```
Comment (axelboc) [inline .prettierignore:4]: Ignored automatically.

File: package.json [hunk 3]
```diff
@@ -43,7 +43,7 @@
     "d3-format": "^3.0.0",
     "d3-scale": "^3.3.0",
     "d3-scale-chromatic": "^2.0.0",
-    "lodash-es": "^4.17.21",
+    "lodash": "^4.17.21",
```
Comment (axelboc) [inline package.json:46]: I double-checked: the size of lodash in the output bundle stays exactly the same.

**Comment by axelboc** [comment]
Hmm, it seems the tests now fail when importing `d3-format`... 🤷 Investigating now.

**Review by loichuder — APPROVED** [review]

**Pull request merged**